src/dex/balancer-v2/balancer-v2-math.ts

```diff
@@ -183,10 +202,8 @@ class LogExpMath {
 
   // Bounds for ln_36's argument. Both ln(0.9) and ln(1.1) can be represented with 36 decimal places in a fixed point
   // 256 bit integer.
-  static LN_36_LOWER_BOUND: bigint =
-    BigInt(LogExpMath.ONE_18) - BigInt('100000000000000000');
-  static LN_36_UPPER_BOUND: bigint =
-    BigInt(LogExpMath.ONE_18) + BigInt('100000000000000000');
+  static LN_36_LOWER_BOUND: bigint = LogExpMath.ONE_18 - BI_POWS[17];
+  static LN_36_UPPER_BOUND: bigint = LogExpMath.ONE_18 + BI_POWS[17];
 
   static MILD_EXPONENT_BOUND: bigint =
     BigInt(2) ** BigInt(254) / LogExpMath.ONE_20;
```

Comment:
Missed replacement

Reply:
Currently I replaced with `n` postfixed, but when I was doing this, I did it intentionally, because we create it once on initialization. Anyway, fixed